ci/docs.sh

```diff
@@ -51,9 +51,12 @@ cp -r ../build/html/* $api_dest/
 
 git add --all
 dff=$(git diff --staged --stat)
+repo_url=$(git config --get remote.origin.url)
+url=${repo_url#https://}
+github_account=${GITHUB_ACCOUNT:-nvauto}
 if [[ -n $dff ]]; then
     git commit -m "${TAG}"
-    git push origin gh-pages
+    git push -f https://${github_account}:${GITHUB_TOKEN}@${url} gh-pages
```

Comment:
Curious where GITHUB_TOKEN is being defined.

Reply:
it will be defined in the internal nightly/release pipelines